src/Data/Quantity/Math.purs

```diff
@@ -82,27 +106,51 @@ ln = lift Decimal.ln
 sin ∷ Quantity → Result
 sin = lift Decimal.sin
 
+csc ∷ Quantity → Result
+csc = lift \x -> one / Decimal.sin x
+
 tan ∷ Quantity → Result
 tan = lift Decimal.tan
 
+cot ∷ Quantity → Result
+cot = lift \x -> one / Decimal.tan x
+
 sinh ∷ Quantity → Result
 sinh = lift Decimal.sinh
 
+csch ∷ Quantity → Result
+csch = lift \x -> one / Decimal.sinh x
+
 cosh ∷ Quantity → Result
 cosh = lift Decimal.cosh
 
+sech ∷ Quantity → Result
+sech = lift \x -> one / Decimal.cosh x
+
 tanh ∷ Quantity → Result
 tanh = lift Decimal.tanh
 
+coth ∷ Quantity → Result
+coth = lift \x -> one / Decimal.tanh x
+
 asinh ∷ Quantity → Result
 asinh = lift Decimal.asinh
 
+acsch ∷ Quantity → Result
+acsch = lift \x -> Decimal.asinh (one / x)
+
 acosh ∷ Quantity → Result
 acosh = lift Decimal.acosh
 
+asech ∷ Quantity → Result
+asech = lift \x -> Decimal.acosh (one / x)
+
 atanh ∷ Quantity → Result
 atanh = lift Decimal.atanh
 
+acoth ∷ Quantity → Result
+acoth = lift \x -> Decimal.atanh (one / x)
```

Comment:
Do you have a reference for those definitions?

Reply:
Well, https://github.com/Qalculate/libqalculate<sup>*</sup>. :) I also checked each function manually.

<sub>* let's pretend I didn't take all the definitions from there</sub>

Reply:
Alright.